libs/coin-modules/coin-sui/src/api/index.test.ts

```diff
@@ -0,0 +1,120 @@
+import { createApi } from "./index";
```

Comment:
Shouldn't the API just have an integration test? Otherwise, with this unit test, you're just mocking a return value and asserting that same value.